views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js

```diff
@@ -45,7 +45,10 @@ define([
         $container.off('.commonRenderer');
 
         $container.on('click.commonRenderer', '.qti-choice', function(e){
-
+            if ($(this).closest('.qti-itemBody').hasClass('js-choice-eliminating')) {
+                return;
```

Comment:
ok, but isn't it better to run this snippet after e.stopPropagation() (line 54) ?